monster-cards/src/widgets/navbar/createNavbar.ts

```diff
@@ -14,33 +14,37 @@ export type NavBarLinkDefinition = {
 	href: string;
 }
 
-function listItem(childNode: DNode): DNode {
-	return d('li', {}, [ childNode ]);
+let counter = 0;
+
+function listItem(childNode: DNode, key?: any): DNode {
+	const options = key ? { key } : {};
+	return d('li', options, [ childNode ]);
 }
 
 function createNavBarLink({ text: innerHTML, href }: NavBarLinkDefinition): DNode {
-	return listItem(d('a', { href, innerHTML }));
+	return listItem(d('a', { key: counter++, href, innerHTML }), innerHTML);
```

Comment:
did this get changed to `key` now? last time I checked we were still using `id` here.